x/mint/module/abci.go

```diff
@@ -0,0 +1,84 @@
+package mint
+
+import (
+	"context"
+	"time"
+
+	cosmosMath "cosmossdk.io/math"
+	"github.com/allora-network/allora-chain/x/mint/keeper"
+	"github.com/allora-network/allora-chain/x/mint/types"
+	"github.com/cosmos/cosmos-sdk/telemetry"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// TODO: MaxSupply should be defined somewhere else
```

Comment:
lets put this in a params object that is pulled in and exported/imported from/to the genesis.json file

can reference genesis.go in the emissions module for how I did that for that module https://github.com/allora-network/allora-chain/blob/383c8369ac6ab1b7dda2fb73b4a442ebb4d0d930/x/emissions/genesis.go#L8